README.md

```diff
@@ -3,27 +3,27 @@
 </h1>
 
 <div align="center">
-  
-  **A simple, small, and content-agnostic modal for [Svelte](https://svelte.dev).**
-  
```

Comment:
Please undo these and the following unnecessary removal of indentations.

Reply:
My bad, guess my autolinter caught em. Will fix!